app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -64,44 +56,6 @@ function InPersonPrepareStep({ toPreviousStep, value }) {
           headingUnstyled
         />
       </ProcessList>
-
-      <hr className="margin-bottom-4" />
-
-      <h2>{t('in_person_proofing.body.prepare.bring_title')}</h2>
-
-      <IconList>
```

Comment:
It looks like this was the only place we were using `IconList` components. Should we consider removing the component implementations?

Reply:
thanks for pointing that out! i think i'll add a separate ticket for checking in on that. I'm not sure if design has use for that component in any future work, and if not it looks like perhaps other components may potentially need to be removed as well (IconListItem, IconListContenntent perhaps)